src/persist-client/src/stats.rs

```diff
@@ -67,38 +67,41 @@ pub(crate) const STATS_BUDGET_BYTES: Config<usize> = Config::new(
     "The budget (in bytes) of how many stats to maintain per batch part.",
 );
 
-pub(crate) const STATS_UNTRIMMABLE_COLUMNS_EQUALS: Config<String> = Config::new(
+pub(crate) const STATS_UNTRIMMABLE_COLUMNS_EQUALS: Config<fn() -> String> = Config::new(
     "persist_stats_untrimmable_columns_equals",
-    concat!(
-        // If we trim the "err" column, then we can't ever use pushdown on a
-        // part (because it could have >0 errors).
-        "err,",
-        "ts,",
-        "receivedat,",
-        "createdat,",
-        // Fivetran created tables track deleted rows by setting this column.
-        //
-        // See <https://fivetran.com/docs/using-fivetran/features#capturedeletes>.
-        "_fivetran_deleted,",
-    ),
+    || {
+        [
+            // If we trim the "err" column, then we can't ever use pushdown on a
+            // part (because it could have >0 errors).
+            "err",
+            "ts",
+            "receivedat",
+            "createdat",
+            // Fivetran created tables track deleted rows by setting this column.
+            //
+            // See <https://fivetran.com/docs/using-fivetran/features#capturedeletes>.
+            "_fivetran_deleted",
+        ]
+        .join(",")
```

Comment:
Not married to this! I mostly wanted to demonstrate the flexibility of the new callable default values. But it does seem plausibly less error prone like this. No chance of missing a comma inside the string literals.